packages/react-client/src/ReactFlightClient.js

```diff
@@ -160,16 +165,19 @@ export function reportGlobalError<T>(
   });
 }
 
-function readMaybeChunk<T>(maybeChunk: Chunk<T> | T): T {
-  if (maybeChunk == null || (maybeChunk: any).$$typeof !== CHUNK_TYPE) {
+function readMaybeChunk<T>(maybeChunk: SomeChunk<T> | T): T {
+  if (maybeChunk == null || !(maybeChunk instanceof Chunk)) {
     // $FlowFixMe
     return maybeChunk;
   }
-  let chunk: Chunk<T> = (maybeChunk: any);
-  if (chunk.status === RESOLVED) {
-    return chunk.value;
+  let chunk: SomeChunk<T> = (maybeChunk: any);
+  if (chunk._status === RESOLVED) {
+    return chunk._value;
+  } else if (chunk._status === PENDING) {
+    // eslint-disable-next-line no-throw-literal
```

Comment:
Btw this seems like a bug in the no-throw-literal lint. It gets confused by the Flow annotation.